inc/Engine/CriticalPath/CriticalCSSSubscriber.php

```diff
@@ -696,8 +603,19 @@ public function async_css( $buffer ) {
 			return $buffer;
 		}
 
-		$noscripts = '';
+		return $this->async_css_tag_update( $tags_match, $buffer );
+	}
 
+	/**
+	 * Updates the HTML code by defering all CSS matches files.
+	 *
+	 * @param  array  $tags_match All matched css files.
+	 * @param  string $buffer     HTML code.
+	 * @return string             Modified HTML code with defer CSS.
+	 */
+	private function async_css_tag_update( $tags_match, $buffer ) {
+		$excluded_css = array_flip( get_rocket_exclude_async_css() );
```

Comment:
`get_rocket_exclude_async_css()` deprecate the function, and add a method in `CriticalCSS` class to replace it. It's not used anywhere else